<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginPage</class>
 <widget class="QWidget" name="LoginPage">
  <layout class="QVBoxLayout">
   <item><spacer><property name="orientation"><enum>Qt::Vertical</enum></property></spacer></item>
   <item alignment="Qt::AlignCenter">
    <widget class="QFrame" name="auth_card">
     <property name="minimumSize"><size><width>420</width><height>0</height></size></property>
     <property name="maximumSize"><size><width>420</width><height>16777215</height></size></property>
     <layout class="QVBoxLayout">
      <property name="spacing"><number>6</number></property>
      <property name="leftMargin"><number>45</number></property>
      <property name="topMargin"><number>50</number></property>
      <property name="rightMargin"><number>45</number></property>
      <property name="bottomMargin"><number>50</number></property>
      <item alignment="Qt::AlignCenter">
       <widget class="QLabel" name="logoLabel">
        <property name="minimumSize"><size><width>80</width><height>80</height></size></property>
        <property name="maximumSize"><size><width>80</width><height>80</height></size></property>
        <property name="text"><string/></property>
        <property name="alignment"><set>Qt::AlignCenter</set></property>
        <property name="scaledContents"><bool>true</bool></property>
       </widget>
      </item>
      <item><spacer><property name="orientation"><enum>Qt::Vertical</enum></property><property name="sizeHint" stdset="0"><size><width>20</width><height>16</height></size></property></spacer></item>
      <item><widget class="QLabel" name="auth_title"><property name="text"><string>Bienvenue</string></property><property name="alignment"><set>Qt::AlignCenter</set></property></widget></item>
      <item><widget class="QLabel" name="auth_subtitle"><property name="text"><string>Connectez-vous pour continuer</string></property><property name="alignment"><set>Qt::AlignCenter</set></property></widget></item>
      <item><spacer><property name="orientation"><enum>Qt::Vertical</enum></property><property name="sizeHint" stdset="0"><size><width>20</width><height>32</height></size></property></spacer></item>
      <item><widget class="QLabel"><property name="text"><string>Nom d'utilisateur</string></property></widget></item>
      <item><widget class="QLineEdit" name="usernameInput"><property name="placeholderText"><string>Entrez votre nom d'utilisateur</string></property></widget></item>
      <item><spacer><property name="orientation"><enum>Qt::Vertical</enum></property><property name="sizeHint" stdset="0"><size><width>20</width><height>16</height></size></property></spacer></item>
      <item><widget class="QLabel"><property name="text"><string>Mot de passe</string></property></widget></item>
      <item><widget class="QLineEdit" name="passwordInput"><property name="echoMode"><enum>QLineEdit::Password</enum></property><property name="placeholderText"><string>Entrez votre mot de passe</string></property></widget></item>
      <item><spacer><property name="orientation"><enum>Qt::Vertical</enum></property><property name="sizeHint" stdset="0"><size><width>20</width><height>24</height></size></property></spacer></item>
      <item><widget class="QPushButton" name="loginButton"><property name="objectName"><string>primary_button</string></property><property name="text"><string>Se connecter</string></property><property name="default"><bool>true</bool></property></widget></item>
      <item><spacer><property name="orientation"><enum>Qt::Vertical</enum></property><property name="sizeHint" stdset="0"><size><width>20</width><height>12</height></size></property></spacer></item>
      <item><widget class="QPushButton" name="registerLink"><property name="objectName"><string>text_button</string></property><property name="text"><string>Pas de compte ? Inscrivez-vous</string></property></widget></item>
     </layout>
    </widget>
   </item>
   <item><spacer><property name="orientation"><enum>Qt::Vertical</enum></property></spacer></item>
  </layout>
 </widget>
</ui>
